librz/debug/p/native/windows/windows_message.c

```diff
@@ -448,18 +448,16 @@ RZ_API void rz_w32_print_windows(RzDebug *dbg) {
 	rz_list_free(windows);
 }
 
-RZ_API bool rz_w32_add_winmsg_breakpoint(RzDebug *dbg, const char *input) {
-	rz_return_val_if_fail(dbg && input, false);
-	char *name = strdup(input);
+RZ_API bool rz_w32_add_winmsg_breakpoint(RzDebug *dbg, const char *arg_name, const char *arg_window_id) {
```

Comment:
I recommend to rename them to just `name` and `window_id`. Also there is no need to `strdup(window_id)`, I think.